src/components/input/input-text/input-text.module.css

```diff
@@ -0,0 +1,17 @@
+.multipleInputsMasked:focus-within span {
+  outline: Highlight auto 0.1rem;
+  outline: -webkit-focus-ring-color auto 0.1rem;
+}
+
+.multipleInputsMasked input {
+  caret-color: transparent;
+  letter-spacing: 3.15rem;
+  width: 27rem;
+}
+
+@media (min-width: 1024px){
+  .multipleInputsMasked input {
+  width: 75%;
+  letter-spacing: 4.85rem;
+ }
+}
```

Comment:
The 6-squares custom appearance input requires very specific value for `width` and `letter-spacing` (`caret-color` isn't in our Tailwind version, but in JIT mode) and forces this media-query in the CSS stylesheet.

For the `letter-spacing` property, I could maybe add these specific values inside the theme and avoid having it in the stylesheet. But for the `width`, I don't really want to touch the default Tailwind config. Wdyt ?  

(I feel like you won't like it @Fenntasy so I ping you here 😄 )

Reply:
JIT? 🤔 Just-in-time compilation ?

Reply:
I'm not against writing some custom CSS for very specific components and, imho, it is the case here (like it was for the buttons).

So, I'm ok with that.